app/controllers/v1/article_controller.rb

```diff
@@ -0,0 +1,6 @@
+class V1::ArticleController < ApplicationController
```

Comment:
Why do you have singular for controller, and two of them? 🤔

Reply:
removed duplicates